docs/blog/2025-03-19-using-angular-with-rspack.md

```diff
@@ -0,0 +1,166 @@
+
+We are thrilled to announce that Angular Rspack is now being maintained by the Nx team and is available for use with Nx. This means that you can now use Rspack with Angular and Nx and enjoy the benefits of both.  
+The new package is called [@nx/angular-rspack](https://www.npmjs.com/package/@nx/angular-rspack) and it is available on npm while the repository has been moved to [nrwl/angular-rspack](https://github.com/nrwl/angular-rspack).
+
+We believe that a large portion of Angular applications still using Webpack will be able to migrate to Angular Rspack without issue.
+
+There are some limitations and missing features that are currently being worked on and on the roadmap to support. They have been listed at the bottom of this article.
+
+## Migrating from Angular Webpack to Angular Rspack
+
+We have worked hard to make the migration process as smooth as possible.
+A new generator has been added to the `@nx/angular` package called `convert-to-rspack` which will help you migrate your Angular applications from Webpack to Rspack.
+
+The steps are very simple:
+
+1. Run `nx migrate latest` to update your workspace to the latest version of Nx.
+2. Run `nx g @nx/angular:convert-to-rspack` to migrate your Angular application to Rspack.
+
+We have also created a [guide](/recipes/angular/rspack/migrate-from-webpack) that walks you through the process step-by-step.
+Even if you're currently using the Angular CLI, it's as simple as first running `npx nx init` in your workspace and then running `npx nx convert-to-rspack`.
+
+## Using Angular Rspack
+
+You'll notice a that your `build` and `serve` targets have been removed from your project, and are instead inferred by the `@nx/rspack/plugin`.
+You should also notice a new `rspack.config.ts` file in your project which looks something like this:
+
+```ts
+import { createConfig } from '@nx/angular-rspack';
+export default createConfig(
+  {
+    options: {
+      root: __dirname,
+      outputPath: {
+        base: '../../dist/apps/app',
+      },
+      index: './src/index.html',
+      browser: './src/main.ts',
+      polyfills: ['zone.js'],
+      tsConfig: './tsconfig.app.json',
+      assets: [
+        './src/favicon.ico',
+        './src/assets',
+        {
+          input: './public',
+          glob: '**/*',
+        },
+      ],
+      styles: ['./src/styles.scss'],
+      scripts: [],
+    },
+  },
+  {
+    production: {
+      options: {
+        outputPath: {
+          base: '../../dist/apps/app-prod',
+        },
+        index: './src/index.prod.html',
+        browser: './src/main.prod.ts',
+        tsConfig: './tsconfig.prod.json',
+      },
+    },
+  }
+);
+```
+
+{% callout type="info" title="createConfig Information" %}
```

Comment:
optional. For these "expanded information blocks" I often use the `type="deepdive"` which is made on purpose for  when you want to go on a little tangent and expand on something. Those are collapsed by default. 

it's optional for this one as it is super short, but you could still use it and even expand a little more about what it does potentially.

Reply:
I didn’t know we had a type deep dive, I like it!